.circleci/config.yml

```diff
@@ -50,22 +50,6 @@ jobs:
             - bin
             - profiles
 
-  lint:
```

Comment:
For reader's convenience: I asked Marko to do this - golangci-lint online service should be enough for us, thus this should be dropped